jenkins/test3_drop_in_envs.sh

```diff
@@ -1,12 +1,43 @@
 #!/usr/bin/env bash
 
 CDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )"/.. && pwd )"
+DRUM_BUILDER_IMAGE="datarobot/drum-builder"
+echo $CDIR
+
+# pull DRUM builder container and build DRUM wheel
+docker pull ${DRUM_BUILDER_IMAGE}
+
+# If we are in terminal will be true when running the script manually. Via Jenkins it will be false
+TERMINAL_OPTION=""
+if [ -t 1 ] ; then
+  TERMINAL_OPTION="-t"
+fi
+
+docker run -i ${TERMINAL_OPTION} --user $(id -u):$(id -g) -v $CDIR:/tmp/drum $DRUM_BUILDER_IMAGE bash -c "cd /tmp/drum/custom_model_runner && make"
+
+DRUM_WHEEL=$(find custom_model_runner/dist/datarobot_drum*.whl)
+DRUM_WHEEL_FILENAME=$(basename $DRUM_WHEEL)
+DRUM_WHEEL_REAL_PATH=$(realpath $DRUM_WHEEL)
+
+# Change every environment Dockerfile to install fresly build DRUM wheel
+pushd public_dropin_environments
+DIRS=$(ls)
+for d in $DIRS
+do
+  pushd $d
+  cp $DRUM_WHEEL_REAL_PATH .
+  echo "" >> Dockerfile
```

Comment:
You can use a multi line doc to write to the Dockerfile..

Reply:
done